doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
+
+<p align="center"><b>Table 2-5:</b> Containerised Platform: LCM Requirements. </p>
+
 
+<a name="2.3.6"></a>
+### 2.3.6 Assurance
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.asr.mon.01` | Integration | The Architecture **must** include integration with various infrastructure components to support collection of telemetry for assurance monitoring and network intelligence. |
+| `req.asr.mon.02` | Monitoring | The Architecture **should** support Network Intelligence capabilities that allow richer diagnostic capabilities which take as input broader set of data across the network and from CNF workloads. |
+| `req.asr.mon.03` | Monitoring | The Architecture **must** allow for the collection and dissemination of performance and fault information. |
+| `req.asr.mon.04` | Network | The NFVI Network Fabric and Network Operating System **must** provide network operational visibility through alarming and streaming telemetry services for operational management, engineering planning, troubleshooting, and network performance optimisation. |
+
+<p align="center"><b>Table 2-6:</b> Containerised Platform: Assurance Requirements.</p>
+
+<a name="2.3.7"></a>
+### 2.3.7 Security
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.sec.gen.01` | General | The Architecture **must** provide workload isolation. |
+| `req.sec.gen.02` | General | The Architecture **must** support policy based RBAC. |
+| `req.sec.gen.03` | General | The Architecture **must** support a centralised authentication and authorisation mechanism. |
+| `req.sec.zon.01` | Zoning | The Architecture **must** support identity management. |
+| `req.sec.zon.02` | Zoning | The Architecture **must** support password encryption. |
+| `req.sec.zon.03` | Zoning | The Architecture **must** support data, at-rest and in-flight, encryption. |
+| `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. |
+| `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. |
+| `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. |
+| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with ISO 27001 and adhere to the V1.1 NIST Cybersecurity Framework. |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with ISO27001. |
```

Comment:
do you mean audit logs?
if not, we should add additional requirements related to audit trails

Reply:
@Levovar Yes -- changed to "... have all security audit logs"